src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -602,6 +604,8 @@ protected virtual void OnVisibilityChanged(Visibility oldValue, Visibility newVi
 		partial void OnIsHitTestVisibleChangedPartial(bool oldValue, bool newValue)
 		{
 			UpdateHitTest();
+
+			UpdateDOMProperties();
```

Comment:
Test for feature configuration at all call sites of `UpdateDOMProperties`. It will avoid a common-case virtual call that does nothing.

Reply:
done